data/brands/amenity/vending_machine.json

```diff
@@ -762,6 +762,16 @@
         "vending": "newspapers"
       }
     },
+    {
+      "displayName": "Teika",
+      "locationSet": {"include": ["es"]},
+      "matchNames": ["teika","Teica"],
```

Comment:
`teika` and `Teika` only have difference in uppercase letter so `teika` don't need to be added. Don't worry, it will be auto removed when formatter run and you don't need to do anything.